internal/sidekick/internal/parser/discovery/methods.go

```diff
@@ -21,32 +21,68 @@ import (
 	"github.com/googleapis/librarian/internal/sidekick/internal/api"
 )
 
-func makeServiceMethods(model *api.API, serviceID string, doc *document, resource *resource) ([]*api.Method, error) {
+func makeServiceMethods(model *api.API, serviceName, serviceID string, doc *document, resource *resource) ([]*api.Method, []*api.Message, error) {
+	requests := &api.Message{
+		Name:          serviceName,
+		ID:            serviceID,
```

Comment:
repeating IDs? :unamused: 

I know why we are doing it. It just gives me the ick.